pgjdbc/src/main/java/org/postgresql/util/PGtokenizer.java

```diff
@@ -47,14 +64,15 @@ public PGtokenizer(String string, char delim) {
   public int tokenize(String string, char delim) {
     tokens.clear();
 
+    final Deque<Character> stack = new ArrayDeque<>();
+
     // nest holds how many levels we are in the current token.
     // if this is > 0 then we don't split a token when delim is matched.
     //
     // The Geometric datatypes use this, because often a type may have others
     // (usualls PGpoint) imbedded within a token.
```

Comment:
this comment should be updated to reflect that stack now maintains levels in the current token.